commit 8846174de239229b423bcfcad031acb59e5df721
Author: Seth Forshee <seth.forshee@canonical.com>
Date:   Wed Aug 30 08:14:06 2017 -0500

    UBUNTU: SAUCE: selftests/powerpc: Disable some ptrace selftests
    
    The ptrace-tm-vsx, ptrace-tm-spd-vsx, and ptrace-tm-spr tests
    FTBFS with the gcc in artful due to inline asm which includes r2
    in the clobber list. Disable these tests until a solution is
    found.
    
    Signed-off-by: Seth Forshee <seth.forshee@canonical.com>

diff --git a/tools/testing/selftests/powerpc/ptrace/Makefile b/tools/testing/selftests/powerpc/ptrace/Makefile
index 480305266504..8072c9e69d3f 100644
--- a/tools/testing/selftests/powerpc/ptrace/Makefile
+++ b/tools/testing/selftests/powerpc/ptrace/Makefile
@@ -1,7 +1,6 @@
 # SPDX-License-Identifier: GPL-2.0
 TEST_PROGS := ptrace-gpr ptrace-tm-gpr ptrace-tm-spd-gpr \
-              ptrace-tar ptrace-tm-tar ptrace-tm-spd-tar ptrace-vsx ptrace-tm-vsx \
-              ptrace-tm-spd-vsx ptrace-tm-spr
+              ptrace-tar ptrace-tm-tar ptrace-tm-spd-tar ptrace-vsx
 
 include ../../lib.mk
 
